admin/javascript/jquery-changetracker/lib/jquery.changetracker.js

```diff
@@ -88,9 +89,31 @@
 
 			// setup original values
 			var fields = this.getFields(), origVal;
-			fields.filter(':radio,:checkbox').bind('click.changetracker', onchange);
-			fields.not(':radio,:checkbox').bind('change.changetracker', onchange);
+
+			// omit radio buttons and checkboxes, along with any other selectors that have a custom change event
+			var omittedSelectors = [':radio', ':checkbox']
+				.concat($.map(options, function(key, val) {
+					if(typeof key === 'string') return key;
+				}))
+				.join(',');
+
+			fields.filter(omittedSelectors).on('click.changetracker', onchange);
+			fields.not(omittedSelectors).on('change.changetracker', onchange);
+			
+			// Handle omitted selectors
 			fields.each(function() {
+				// For selectors with custom change events
+				for(var selector in options.customChangeEvents) {
+					var eventName;
+					if(options.customChangeEvents.hasOwnProperty(selector)) {
```

Comment:
How could this be false? Isn't it a property by virtue of the loop on options.customChangeEvents?